app/scenes/channel_info/channel_info.js

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PropTypes, PureComponent} from 'react';
+import {
+    ScrollView,
+    StyleSheet,
+    View
+} from 'react-native';
+
+import ChannelInfoHeader from './channel_info_header';
+import ChannelInfoRow from './channel_info_row';
+
+const style = StyleSheet.create({
+    container: {
+        flex: 1
+    },
+    footer: {
+        marginTop: 40
+    },
+    separator: {
+        flex: 1,
+        marginHorizontal: 15
+    },
+    separatorContainer: {
+        flex: 1,
+        backgroundColor: '#fff',
+        height: 1
+    },
+    scrollView: {
+        flex: 1
+    }
+});
+
+export default class Channel extends PureComponent {
+    static propTypes = {
+        currentChannel: PropTypes.object.isRequired,
+        currentChannelCreatorName: PropTypes.string,
+        currentChannelMemberCount: PropTypes.number,
+        currentChannelUrl: PropTypes.string.isRequired,
+        isFavorite: PropTypes.bool.isRequired,
+        theme: PropTypes.object.isRequired,
+        actions: PropTypes.shape({
+            getChannelStats: PropTypes.func.isRequired
+        })
+    }
+
+    componentDidMount() {
+        this.props.actions.getChannelStats(this.props.currentChannel.team_id, this.props.currentChannel.id);
+    }
+
+    render() {
+        const {
+            currentChannel,
+            currentChannelCreatorName,
+            currentChannelMemberCount,
+            currentChannelUrl,
+            isFavorite,
+            theme
+        } = this.props;
+
+        // Would like some feedback on this...
+        const actionRows = [{
+            action: () => true,
```

Comment:
i'm assuming we still need to add the functionality for each of this actions here, but that's another ticket right?